<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapPage</class>
 <widget class="QWidget" name="MapPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_root">
   <property name="spacing"><number>12</number></property>
   <property name="leftMargin"><number>12</number></property>
   <property name="topMargin"><number>12</number></property>
   <property name="rightMargin"><number>12</number></property>
   <property name="bottomMargin"><number>12</number></property>

   <item>
    <widget class="QGroupBox" name="group_map">
     <property name="title"><string>Robot Path Visualization</string></property>
     <layout class="QVBoxLayout" name="verticalLayout_map">
      <item>
       <widget class="QGraphicsView" name="mapView">
        <property name="dragMode"><enum>QGraphicsView::ScrollHandDrag</enum></property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_mapButtons">
        <item><widget class="QPushButton" name="btn_zoom_out"><property name="text"><string>-</string></property></widget></item>
        <item><widget class="QPushButton" name="btn_zoom_in"><property name="text"><string>+</string></property></widget></item>
        <item><widget class="QPushButton" name="btn_fit"><property name="text"><string>FIT</string></property></widget></item>
        <item><spacer name="spacer_map"><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
        <item><widget class="QLabel" name="lbl_zoom"><property name="text"><string>Zoom</string></property></widget></item>
        <item>
         <widget class="QSlider" name="sld_zoom">
          <property name="orientation"><enum>Qt::Horizontal</enum></property>
          <property name="minimum"><number>20</number></property>
          <property name="maximum"><number>300</number></property>
          <property name="value"><number>100</number></property>
         </widget>
        </item>
        <item><widget class="QLabel" name="lbl_zoom_value"><property name="text"><string>100%</string></property></widget></item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="group_timeline">
     <property name="title"><string>Controls</string></property>
     <layout class="QVBoxLayout" name="verticalLayout_timeline">
      <property name="spacing"><number>10</number></property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_mode">
        <item><widget class="QLabel" name="lbl_mode"><property name="text"><string>Mode</string></property></widget></item>
        <item>
         <widget class="QComboBox" name="cmb_mode">
          <item><property name="text"><string>Full</string></property></item>
          <item><property name="text"><string>Recent (Tail)</string></property></item>
          <item><property name="text"><string>Playback</string></property></item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_clear">
        <item><widget class="QPushButton" name="btn_clear_path"><property name="text"><string>Clear Trace</string></property></widget></item>
        <item><widget class="QLabel" name="lbl_poses"><property name="text"><string>0 pts</string></property></widget></item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_tail">
        <item><widget class="QLabel" name="lbl_tail"><property name="text"><string>Tail(s)</string></property></widget></item>
        <item>
         <widget class="QSlider" name="sld_tail_sec">
          <property name="orientation"><enum>Qt::Horizontal</enum></property>
          <property name="maximum"><number>3600</number></property>
          <property name="value"><number>300</number></property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_playback">
        <item><widget class="QLabel" name="lbl_playback"><property name="text"><string>Playback</string></property></widget></item>
        <item><widget class="QCheckBox" name="chk_playback"><property name="text"><string/></property></widget></item>
        <item><spacer name="spacer_pb"><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
        <item><widget class="QLabel" name="lbl_pb_idx"><property name="text"><string>0 / 0</string></property></widget></item>
       </layout>
      </item>
      <item><spacer name="spacer_bottom"><property name="orientation"><enum>Qt::Vertical</enum></property></spacer></item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
</ui>